waiter/test/waiter/scheduler/marathon_test.clj

```diff
@@ -884,19 +866,437 @@
+                                            (is (= marathon-api in-marathon-api))
+                                            (is (= service-id in-service-id))
+                                            (is (= {:cmd "tc" :cpus 2 :id service-id :instances 15 :mem 4} descriptor))
+                                            (deliver updated-invoked-promise :invoked))]
+          (scheduler/scale-app marathon-scheduler service-id instances true)
+          (is (= :invoked (deref deleted-deployment-promise 0 :not-invoked)))
+          (is (= :invoked (deref updated-invoked-promise 0 :not-invoked))))))
+
+    (testing "forced scale of service - more instances"
+      (let [updated-invoked-promise (promise)
+            deleted-deployment-promise (promise)
+            instances 20
+            deployment-id "d-1234"]
+        (with-redefs [extract-service-deployment-info (fn [in-marathon-api in-service-id]
+                                                        (is (= marathon-api in-marathon-api))
+                                                        (is (= service-id in-service-id))
+                                                        {:id deployment-id})
+                      marathon/delete-deployment (fn [in-marathon-api in-deployment-id]
+                                                   (is (= marathon-api in-marathon-api))
+                                                   (is (= deployment-id in-deployment-id))
+                                                   (deliver deleted-deployment-promise :invoked))
+                      marathon/get-app (fn [in-marathon-api in-service-id]
+                                         (is (= marathon-api in-marathon-api))
+                                         (is (= service-id in-service-id))
+                                         {:app {:cmd "tc" :cpus 2 :id service-id :instances 25 :mem 4
+                                                :tasks (->> (fn [] {:appId service-id
+                                                                    :id (str service-id "." (rand-int 1000))})
+                                                            (repeatedly 15))}})
+                      marathon/update-app (fn [in-marathon-api in-service-id descriptor]
+                                            (is (= marathon-api in-marathon-api))
+                                            (is (= service-id in-service-id))
+                                            (is (= {:cmd "tc" :cpus 2 :id service-id :instances instances :mem 4} descriptor))
+                                            (deliver updated-invoked-promise :invoked))]
+          (scheduler/scale-app marathon-scheduler service-id instances true)
+          (is (= :invoked (deref deleted-deployment-promise 0 :not-invoked)))
+          (is (= :invoked (deref updated-invoked-promise 0 :not-invoked))))))))
+
+(defmacro launch-sync-deployment-maintainer
```

Comment:
We don't need to start the timer task for unit tests.

Reply:
Done.